BetterMD/elements/table.py

```
@@ -426,40 +501,47 @@ def data(self):
+        def op_get(list:'list', i:'int', d):
+            try:
+                return list[i]
+            except:
+                return d
+
+        self.header = op_get(self.table.headers, len(self.row.data) - 1, HeadlessTd())
+        if isinstance(self.header, HeadlessTd):
+            self.table.cols[self.header] = [self]
+        else:
+            self.table.cols[self.header].append(self)
+        return super().prepare(parent, dom, table=table, data=self, *args, **kwargs)
```

Ruff (0.8.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)